server/tests/test_multiple_recceiver.py

```diff
@@ -27,16 +27,16 @@ class TestMultipleRecceiver:
     def test_number_of_channels_and_channel_name(self, cf_client: ChannelFinderClient) -> None:
         channels = cf_client.find(name="*")
         assert len(channels) == EXPECTED_DEFAULT_CHANNEL_COUNT
-        assert channels[0]["name"] == "IOC1-1:ai:archive"
+        assert channels[0]["name"] == DEFAULT_CHANNEL_NAME
```

Comment:
Is there a guarantee that the records will be returned from CF in a particular order? It seems potentially fragile to test that you expect 8 channels and that the first one has a specific name.

Reply:
So if the record is renamed this might break 🙃